src/compositions/useLink.ts

```diff
@@ -106,24 +119,43 @@ export function useLink(
       routerPrefetch,
       linkPrefetch,
     })
+
+    props = prefetchPropsForRoute(route, {
+      getRouteProps,
```

Comment:
why pass this through? can't the `prefetchPropsForRoute` below just call it directly from the import?